lib/testing/src/lib/core/pages/form/form-fields.ts

```diff
@@ -178,6 +180,11 @@ export class FormFields {
         return widgetReadOnly;
     }
 
+    async isFormFieldEnabled(formFieldId: string): Promise<boolean> {
+        const formField = element(by.css(`input[id=${formFieldId}]`));
```

Comment:
`input[id=${formFieldId}]` can be simplified into `input#${formFieldId}`

Reply:
Also, since an `id` is unique, you should be able to use just 

    by.id(`${formFieldId}`)

Reply:
Done. Thanks 😄